config/examples/skaffold.yaml

```diff
@@ -139,6 +139,22 @@ collections:
       - milestone-pr-needs-merge
       - milestone-recently-closed
 
+  - id: planning
+    name: Planning
+    description: >
+      Skafold Team Planning Board.
+
+      This board is used by Skaffold team to plan upcoming releases.
+      The timeline and priority of issues listed here are subject to change.
+
+    display: planning
+    overflow: 3
```

Comment:
The planning dashboard doesn't appear to use overflow. Remove this to avoid confusion.